src/pages/cart/CartUI.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import CartItem from './CartItem';
+
+const tableHeader = [
+  'Product',
+  'Info',
+  'Price',
+  'Qty.',
+  'Mileage',
+  'Delivery',
+  'Charge',
+  'Total',
+  '선택',
+];
+
+const tableFooter = [
+  { title: 'Price' },
+  { title: 'Shipping' },
+  { title: 'Total' },
+];
+
+const CartUI = ({ cartList, itemTotal }) => {
+  const price = cartList.map(item => item.price).reduce((a, b) => a + b, 0);
+  const shipping = price > 50000 ? 0 : 3000;
+  const total = price + shipping;
+
+  const priceArr = [price, shipping, total];
+
+  return (
+    <>
+      <table>
+        <thead>
+          <tr>
+            <th className="checkBox">
+              <input type="checkbox" />
+            </th>
+            {tableHeader.map(row => {
+              return <th key={row}>{row}</th>;
+            })}
```

Comment:
```suggestion
            {tableHeader.map(row => <th key={row}>{row}</th>)}
```
arrow function에서 바로 return 할 경우에는 중괄호 및 return 키워드 생략 가능합니다!

Reply:
넵!